 error: write_str_simd not supported on the current architecture
    --> .../value-trait-0.6.1/src/generator.rs:370:9
     |
 370 |         compile_error!("write_str_simd not supported on the current architecture")
     |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 error: could not compile `value-trait` (lib) due to previous error

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -96,6 +96,9 @@ nix = { version = "0.26", default-featur
 ] }
 atty = "0.2"
 
+[target.'cfg(not(any(target_arch = "aarch64", target_arch = "x86_64")))'.dependencies]
+value-trait = { version = "0.6.1", features = ["allow-non-simd"] }
+
 [dev-dependencies]
 nu-test-support = { path = "./crates/nu-test-support", version = "0.82.0" }
 tempfile = "3.5"
